test/e2e/provisioning/helper.go

```diff
@@ -97,56 +99,46 @@ func doesSenarioSelectorMatch(selector *scenarioSelector, testCase scenario) boo
 	return false
 }
 
-func runScenarios(t *testing.T, excludeSelector *scenarioSelector, testParams string, manifestPath string, cloudProvider string) {
+func runScenarios(st *testing.T, excludeSelector *scenarioSelector, testParams []string, manifestPath string, cloudProvider string) {
+	buildNum := os.Getenv("CIRCLE_BUILD_NUM")
```

Comment:
do you think we could change this to read it from `-args` ?

Reply:
Yes will do.